app/src/main/java/br/org/cesar/discordtime/stickysessions/app/StickySessionApplication.kt

```diff
@@ -0,0 +1,83 @@
+package br.org.cesar.discordtime.stickysessions.app
+
+import android.app.Application
+
+import com.crashlytics.android.Crashlytics
+
+import br.org.cesar.discordtime.stickysessions.injectors.components.DaggerListSessionComponent
+import br.org.cesar.discordtime.stickysessions.injectors.components.DaggerLobbyComponent
+import br.org.cesar.discordtime.stickysessions.injectors.components.DaggerLoginComponent
+import br.org.cesar.discordtime.stickysessions.injectors.components.DaggerMeetingComponent
+import br.org.cesar.discordtime.stickysessions.injectors.components.DaggerSessionComponent
+import br.org.cesar.discordtime.stickysessions.injectors.modules.ContextModule
+import br.org.cesar.discordtime.stickysessions.ui.list.ListSessionsActivity
+import br.org.cesar.discordtime.stickysessions.ui.lobby.LobbyActivity
+import br.org.cesar.discordtime.stickysessions.ui.login.LoginActivity
+import br.org.cesar.discordtime.stickysessions.ui.meeting.MeetingActivity
+import br.org.cesar.discordtime.stickysessions.ui.session.SessionActivity
+import io.fabric.sdk.android.Fabric
+
+open class StickySessionApplication : Application() {
+
+    protected var mLoginComponentBuilder: DaggerLoginComponent.Builder
+    protected var mLobbyComponentBuilder: DaggerLobbyComponent.Builder
+    protected var mSessionComponentBuilder: DaggerSessionComponent.Builder
+    protected var mSessionListBuilder: DaggerListSessionComponent.Builder
+    protected var mMeetingComponentBuilder: DaggerMeetingComponent.Builder
+
+    override fun onCreate() {
+        super.onCreate()
+        configureLoginInjectorBuilder()
+        configureMainInjectorBuilder()
+        configureSessionInjectorBuilder()
+        configureSessionListInjectorBuilder()
+        configureMeetingInjectorBuilder()
```

Comment:
Do you think better to use Lazy initialization in each build? each build is just instanced in the first use instead of loading all.

e.g: 
protected var mLoginComponentBuilder: DaggerLoginComponent.Builder by lazy {
    DaggerLoginComponent.builder()
}